src/client/src/pages/detail/detailHeader.tsx

```diff
@@ -84,18 +85,14 @@ const DetailHeader = ({
     history.push("/");
   };
 
-  const handleExport = () => {
+  const handleJsonExport = () => {
     const jsonString = JSON.stringify([borehole], null, 2);
-    const blob = new Blob([jsonString], { type: "application/json" });
-    const url = URL.createObjectURL(blob);
-    const link = document.createElement("a");
-    link.href = url;
-    const name_without_spaces = borehole.alternateName.replace(/\s/g, "_");
-    link.download = `${name_without_spaces}.json`;
-    document.body.appendChild(link);
-    link.click();
-    link.remove();
-    URL.revokeObjectURL(url);
+    downloadData(jsonString, borehole.name.replace(/\s/g, "_"), "application/json");
+  };
+
+  const handleCSVExport = async () => {
+    const csvData = await exportCSVBorehole([borehole.id]);
+    downloadData(csvData, borehole.name.replace(/\s/g, "_"), "text/csv");
```

Comment:
PP: `borehole.name.replace(/\s/g, "_")` in Funktion mit aussagekräftigem Namen auslagern.